base_layer/core/src/test_helpers/mod.rs

```diff
@@ -54,13 +54,13 @@ use crate::{
 #[macro_use]
 mod block_spec;
 pub mod blockchain;
-pub type TestKeyManager = CoreKeyManagerHandle<KeyManagerSqliteDatabase<DbConnection>>;
+pub type MemoryDbCoreKeyManagerHandle = CoreKeyManagerHandle<KeyManagerSqliteDatabase<DbConnection>>;
```

Comment:
I much prefer the older name, this name might infer the proper type, but for 99% of tests, just seeing testkeymanager as the type is much simpler to read and see what you use it for